src/components/EmojiSuggestions/index.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import {View, Pressable} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import {FlatList} from 'react-native-gesture-handler';
```

Comment:
I'm confused why we're importing FlatList from `react-native-gesture-handler` 😕

Reply:
This Flatlist enhances the handling of gestures, particularly in this case we have multiple scrolls occur within the same area and direction

Reply:
Cool. Can you add a note? Also should we use the same `FlatList` component everywhere across the app for better consistency?

Reply:
Unless there is some specific case with gesture capture, then yes, in other cases, there is no need.
The main difference is that it uses a native thread instead of the JS thread to perform a gesture.
I'm adding a note

Reply:
I was asking if we should replace:

```diff
- import {FlatList} from 'react-native';
+ import {FlatList} from 'react-native-gesture-handler';
```

everywhere? i.e: it seems like there are some cases where using `react-native-gesture-handler` fixes some bugs. It looks like a lightweight wrapper around the upstream FlatList, so is there any downside to using it everywhere for consistency?

Reply:
Earlier, it was a minor issue with performance as it builds on top of the standard FlatList component from RN For example, with the swipeable component. It is not significant, but I think using the standard FlatList component from RN is better.